docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -15,7 +15,7 @@ import ReleaseMatrix from '/docs/snippets/_private-operator-release-matrix.mdx';
 This guide provides information for setting up the UID2 Operator Service in [Confidential Space](https://cloud.google.com/confidential-computing#confidential-space), a confidential computing option from [Google Cloud](https://cloud.google.com/docs/overview/) Platform. Confidential Space offers a secure enclave environment, known as a Trusted Execution Environment (TEE).
 
 :::note
- UID2 Private Operator for GCP is not supported in these areas: Europe, China.
+ UID2 <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link> for GCP is not supported in these areas: Europe, China.
```

Comment:
I feel like a supportability note is not the right place to put a link to explanation of what a private operator is. Could the link be somehow encorporated into the normal paragraph above?

Reply:
Done.